### ISSUE TITLE
Space in user path prevents executable from being called.

### ISSUE THREAD

**Issue by legowerewolf** [vercel/vercel#2143]
System:
* Windows 10 Education v1809, build 17763.437
* Node v11.14.0
* Npm v6.7.0

My global npm path has a space in it. When I try calling `now`, this happens:
```shell
D:\Duncan Gibson\Development\GitHub\legowerewolf\website>now
'C:\Users\Duncan' is not recognized as an internal or external command,
operable program or batch file.
```

I `where now`'d, found the `now.cmd` file being called, and fixed it to handle the space properly.
It went from this: `@%~dp0\node_modules\now\download\dist\now.exe %*`
to this: `@"%~dp0node_modules\now\download\dist\now.exe" %*`
and now works just fine. (hey! a pun!)

Do what you like with this, but I'd like to not have to rediscover this the next time I update now.

**Comment by legowerewolf** [comment]
Linked to #2144.

**Comment by thasophearak** [comment]
I close this issue due to this has been fixed in PR #2144 

@legowerewolf  🙌 Thank for the PR.
